plugins/gcpaudit/pkg/gcpaudit/gcpaudit.go

```diff
@@ -30,7 +30,7 @@ const (
 	PluginName               = "gcpaudit"
 	PluginDescription        = "Read GCP Audit Logs"
 	PluginContact            = "github.com/falcosecurity/plugins"
-	PluginVersion            = "0.3.3"
+	PluginVersion            = "0.3.4"
```

Comment:
```suggestion
	PluginVersion            = "0.4.0"
```

Since this is an addition.